core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -311,7 +311,7 @@ public Long call() throws Exception {
   @ReturnType("java.lang.Long")
   @Deprecated
   public Response getUptimeMs() {
-    return RestUtils.call(() -> mWorkerProcess.getUptimeMs());
+    return RestUtils.call(()-> mWorkerProcess.getStartTimeMs());
```

Comment:
is this correct?
should still be `getUptimeMs` right?

Reply:
Sorry, I made mistakes. I think I need pull a new request.